roles/edpm_libvirt/tasks/configure.yml

```diff
@@ -27,21 +33,41 @@
   become: true
   ansible.builtin.template:
     src: "{{ item.src }}"
-    dest: "/etc/libvirt/{{ item.dest }}"
-    mode: "0640"
+    dest: "/etc/{{ item.dest }}"
+    mode: "{{ item.mode | default('0640') }}"
     # FIXME: update to libvirt user/group
     owner: "root"
     group: "root"
   loop:
-    - {"src": "virtlogd.conf", "dest": "virtlogd.conf"}
-    - {"src": "virtnodedevd.conf", "dest": "virtnodedevd.conf"}
-    - {"src": "virtproxyd.conf", "dest": "virtproxyd.conf"}
-    - {"src": "virtqemud.conf", "dest": "virtqemud.conf"}
-    - {"src": "qemu.conf.j2", "dest": "qemu.conf"}
-    - {"src": "virtsecretd.conf", "dest": "virtsecretd.conf"}
+    - {"src": "virtlogd.conf", "dest": "libvirt/virtlogd.conf"}
+    - {"src": "virtnodedevd.conf", "dest": "libvirt/virtnodedevd.conf"}
+    - {"src": "virtproxyd.conf", "dest": "libvirt/virtproxyd.conf"}
+    - {"src": "virtqemud.conf", "dest": "libvirt/virtqemud.conf"}
+    - {"src": "qemu.conf.j2", "dest": "libvirt/qemu.conf"}
+    - {"src": "virtsecretd.conf", "dest": "libvirt/virtsecretd.conf"}
+    - {"src": "auth.conf", "dest": "libvirt/auth.conf", "mode": "0600"}
+    - {"src": "sasl_libvirt.conf", "dest": "sasl2/libvirt.conf"}
   notify:
     - Restart libvirt
 
+- name: Configure libvirt sasl credentials
+  tags:
+    - configure
+    - libvirt
+  become: true
+  block:
+    - name: Add libvirt sasl password
+      ansible.builtin.command:
+        cmd: saslpasswd2 -f /etc/libvirt/passwd.db -p -a libvirt -u overcloud migration
```

Comment:
why overcloud? we do not use that term anymore

should it not be nova or livbirt?

Reply:
see other comment